### ISSUE TITLE
[Feature Request]: Windows 11 support

### ISSUE THREAD

**Issue by VimDiezel** [sashaboy779/KomoSwitch#2]
How do I use it on Windows 11? As far as I know, there is no toolbar in Windows 11. If you know how to use it on Windows 11, please let me know.

**Comment by sashaboy779** [comment]
Yes, there is no custom toolbars in Win 11, so it won't work out of the box.

However, there is a way to bring toolbar support by using [Taskbar Patchers ](https://github.com/valinet/ExplorerPatcher) to replace Win 11 taskbar with Win 10 taskbar, but in my experience, KomoSwitch doesn't work properly with this patcher
